pkg/policies/controlplane/validator_test.go

```diff
@@ -126,15 +126,30 @@ data:
             out_ports:
               output:
                 signal_name: "LATENCY_EMA"
+        - constant:
+            value: "1.1"
+            out_ports:
+              output:
+                signal_name: "EMA_SETPOINT_MULTIPLIER"
+        - arithmetic_combinator:
+            operator: "mul"
+            in_ports:
+              lhs:
+                signal_name: "LATENCY"
```

Comment:
Need to multiply with `LATENCY_EMA` instead of `LATENCY`

Reply:
good catch!